packages/space-opera/src/components/mobile_view/open_mobile_view.ts

```diff
@@ -295,35 +294,20 @@ export class OpenMobileView extends ConnectedLitElement {
         View QR Code
       </mwc-button>
       <mwc-button unelevated icon="cached" @click=${this.postInfo} 
-        ?disabled=${this.isSendingData}
+        ?disabled=${!this.haveReceivedResponse || this.isSendingData}
         style="--mdc-theme-primary: ${outOfSyncColor}">
         Refresh Mobile
       </mwc-button>
-      ${
-        isOutOfSync ? html`
-        <div style="color: #DC143C; margin-top: 5px;">
-          Your mobile view is out of sync with the editor.
-        </div>` :
-                      html``}
-      ${
-        this.isSendingData ? html`
-        <div style="color: white; margin-top: 5px;">
-          Sending data to mobile device... Textured models will take some time.
-        </div>` :
-                             html``}
+      ${this.optionalMessage}
     </div>
-
     <div style="font-size: 14px; font-weight: 500; margin: 16px 0px 10px 0px;">AR Settings:</div>
-    <me-dropdown
-        .selectedIndex=${this.selectedArMode}
-        slot="content" style="width: 71%;"
-        @select=${this.onSelectArMode}
+    <me-checkbox 
+      id="ar-modes" 
+      label="Default AR Modes to Scene Viewer"
```

Comment:
This label may be better: The default places webxr in front of scene-viewer and if someone checks this checkbox, we want scene-viewer to become the default: maybe just, "Default to Scene Viewer" possibly

Reply:
How about "Default AR to Scene Viewer"?